src/test/java/Tests/object/BCommentTable.java

```diff
@@ -42,6 +46,14 @@ public boolean isNewCommentTextCorrect(String name) {
         }
     }
 
+    public void commentNames() {
```

Comment:
This method is never used. Please remove it.

Reply:
done